lang/rs/cndprs/src/bindings/bindings.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2020-2022 Intel Corporation.
+ */
+
+#ifndef _BINDINGS_H_
+#define _BINDINGS_H_
+
+/**
+ *
+ * Wrapper functions for CNDP inline functions required for Rust Bindgen tool.
+ *
+ */
+
+#include <stdint.h>         // for uint64_t, uint32_t, uint8_t
+#include <pktmbuf.h>        // for pktmbuf_pool_create, pktmbuf_info_t
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+uint16_t _pktdev_rx_burst(uint16_t lport_id, pktmbuf_t **rx_pkts, const uint16_t nb_pkts);
```

Comment:
Not a fan of _pktdev_rx_burst() having the _ in front of the names seems misleading to me.  Maybe pktdev_rx_burst_fn() or something like it.

Reply:
OK, we can leave it this way.